app/src/main/java/com/example/concordia_campus_guide/Models/Floors.java

```diff
@@ -15,5 +15,11 @@ public Floors(){
         return Floors;
     }
 
-
+    public List<Place> getPlaces(){
+        List<Place> toReturn = new ArrayList<Place>();
```

Comment:
Here too, var name could be changed to "floors"

Reply:
ok